src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```

Comment:
Yeah, not my favorite part. I can't use the Permission constants in the template. This was the best idea I had to work around it for now